src/plot_rasters.jl

```diff
@@ -0,0 +1,114 @@
+function _balance_grid(nplots)
+    ncols = (nplots - 1) ÷ ceil(Int, sqrt(nplots)) + 1
+    nrows = (nplots - 1) ÷ ncols + 1
+    return ncols, nrows
+end
+
+theme_rasters = Theme(
```

Comment:
Ideally, this should be a Function.  Also, is it necessary to change e.g. font, tick size, etc here?

Reply:
This is how the main themes in Makie are defined isn't?

Reply:
And plan is to have a nice theme for Rasters, this the first iteration 😉

Reply:
No, generally they are also defined as functions (to avoid someone accidentally changing the default attributes, and to avoid global variables).

Would you like to add this to MakieThemes instead of here, so we can start collecting all the themes of Makie in that repo?

Reply:
Oh, maybe? I mean do we want to add another dependency? Maybe not. Transfer the theme there should be OK, but having solo here should be fine, and the convention, I think.